
This patch modifies look&feel of:

 -watchinprogress percentage and progress bar in recordings list window
 -recording progress in schedule window


diff --speed-large-files --no-dereference --minimal -Naur Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/recordings-ui.xml Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/recordings-ui.xml
--- Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/recordings-ui.xml	2022-04-26 01:09:17.000000000 +0200
+++ Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/recordings-ui.xml	2022-08-14 11:42:43.906635589 +0200
@@ -26,20 +26,43 @@
         <buttonlist name="recordings" from="baseitemlist">
             <align>top,hcenter</align>
             <layout>vertical</layout>
-            <area>25,500,740,515</area>
-            <buttonarea>0,0,740,515</buttonarea>
+            <area>25,500,740,505</area>
+            <buttonarea>0,0,740,505</buttonarea>
             <showarrow>no</showarrow>
             <scrollstyle>free</scrollstyle>
             <wrapstyle>items</wrapstyle>
             <arrange>fixed</arrange>
             <showarrow>no</showarrow>
             <statetype name="buttonitem">
+                <area>0,0,100%,50</area>
                 <state name="active">
-                    <area>0,0,100%,56</area>
+                    <progressbar name="buttonprogress1">
+                        <area>0,0,100%,100%</area>
+                        <layout>horizontal</layout>
+                        <style>reveal</style>
+                        <shape name="progressimage">
+                            <area>0,50,100%,1</area>
+                            <type>box</type>
+                            <fill color="#FFFFFF" alpha="128"/>
+                        </shape>
+                    </progressbar>
+                    <progressbar name="buttonprogress2">
+                        <area>0,0,100%,100%</area>
+                        <layout>horizontal</layout>
+                        <style>reveal</style>
+                        <shape name="progressimage">
+                            <area>0,7,100%,90%</area>
+                            <type>box</type>
+                            <fill color="#E0991A" alpha="64"/>
+                        </shape>
+                    </progressbar>
                     <textarea name="titlesubtitle" from="baseitemtext">
                         <area>5,0,-45,100%</area>
-                        <alpha>100</alpha>
-                        <template>%TITLE| %%~ |00x00| %%~ |SUBTITLE%</template>
+                        <alpha>100</alpha>
+                        <!--<template>%TITLE| %%~ |00x00| %%~ |SUBTITLE%</template>-->
+                        <!--<template>%titlesubtitle%% (|progresspercent|%)%</template>-->
+                        <template>%TITLE| %%~ |SUBTITLE%</template>
+                        <font state="running">basefont_l_running</font>
                         <font state="error">damagedfont</font>
                     </textarea>
                     <!-- display watched state only when there is no job running -->
@@ -61,16 +84,6 @@
                         <alpha>255</alpha>
                         <cutdown>no</cutdown>
                     </textarea>
-                    <progressbar name="buttonprogress2" from="baseprogress_xs">
-                        <shape name="progressimage">
-                            <fill color="#4B0000" alpha="255"/>
-                        </shape>
-                    </progressbar>
-                    <progressbar name="buttonprogress1" from="baseprogress_xs">
-                        <shape name="progressimage">
-                            <fill color="#004B00" alpha="255"/>
-                        </shape>
-                    </progressbar>
                     <!-- display watched state only when there is no job running -->
                     <statetype name="watched" depends="!jobstate&amp;!playlist">
                         <alpha>255</alpha>
@@ -89,16 +102,13 @@
         <buttonlist name="groups" from="baseitemlist">
             <align>top,hcenter</align>
             <layout>vertical</layout>
-            <area>25,500,740,515</area>
-            <buttonarea>0,0,740,515</buttonarea>
+            <area>25,500,740,505</area>
+            <buttonarea>0,0,740,505</buttonarea>
             <showarrow>no</showarrow>
             <scrollstyle>free</scrollstyle>
             <wrapstyle>items</wrapstyle>
             <arrange>fixed</arrange>
             <statetype name="buttonitem">
-                <state name="active">
-                    <area>0,0,100%,56</area>
-                </state>
                 <state name="inactive" from="active">
                     <alpha>0</alpha>
                 </state>
@@ -141,7 +151,7 @@
             </textarea>
             <textarea name="subtitle" from="basetext_l">
                 <area>0, 45, 700, 50</area>
-                <alpha>150</alpha>
+		<alpha>150</alpha>
                 <template>%00x00| ~ %%SUBTITLE%</template>
             </textarea>
             <group name="smalldescription" depends="subtitle">
@@ -171,7 +181,7 @@
                     <filename>images/icons/Clock.png</filename>
                 </imagetype>
                 <textarea name="startdate" from="basetext_s">
-                    <area>50,40,100%-115,32</area>
+                    <area>50,40,650,32</area>
                     <alpha>150</alpha>
                     <template>%startdate%%| year%, %starttime% - %endtime% (%lenmins%)</template>
                 </textarea>
@@ -180,33 +190,11 @@
                     <filename>images/icons/Storage.png</filename>
                 </imagetype>
                 <textarea name="storagegroup" from="basetext_s">
-                    <area>50,80,100%-115,32</area>
+                    <area>50,80,650,32</area>
                     <alpha>150</alpha>
                     <template>%filesize_str% of %storagegroup% Storage</template>
                 </textarea>
 
-                <imagetype name="partrecorded" from="baseicon" depends="recordedpercent">
-                    <position>100%-32,40</position>
-                    <filename>images/icons/Record.png</filename>
-                    <alpha>150</alpha>
-                </imagetype>
-                <textarea name="recordedpercent" from="basetext_s">
-                    <area>100%-115,40,100%-40,32</area>
-                    <alpha>150</alpha>
-                    <align>right,vcenter</align>
-                    <template>%|recordedpercent|%%</template>
-                </textarea>
-                <imagetype name="partwatched" from="baseicon" depends="watchedpercent">
-                    <position>100%-32,80</position>
-                    <filename>images/icons/Watched.png</filename>
-                    <alpha>150</alpha>
-                </imagetype>
-                <textarea name="watchedpercent" from="basetext_s">
-                    <area>100%-115,80,100%-40,32</area>
-                    <alpha>150</alpha>
-                    <align>right,vcenter</align>
-                    <template>%|watchedpercent|%%</template>
-                </textarea>
                 <statetype name="watched" from="basewatchedstate">
                     <position>100%-32,80</position>
                 </statetype>
diff --speed-large-files --no-dereference --minimal -Naur Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/schedule-ui.xml Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/schedule-ui.xml
--- Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/schedule-ui.xml	2022-04-26 01:09:17.000000000 +0200
+++ Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/schedule-ui.xml	2022-08-14 11:35:46.387994004 +0200
@@ -469,42 +469,41 @@
             <area>25,505,900,505</area>
             <statetype name="buttonitem">
                 <state name="active">
+                    <progressbar name="buttonprogress2">
+                        <area>0,0,100%,100%</area>
+                        <layout>horizontal</layout>
+                        <style>reveal</style>
+                        <shape name="progressimage">
+                            <area>50,7,100%,90%</area>
+                            <type>box</type>
+                            <fill color="#E0991A" alpha="64"/>
+                        </shape>
+                    </progressbar>
                     <textarea name="card" from="baseitemtext">
                         <area>5,0,50,100%</area>
                         <font state="error">basefont_l_bad</font>
-                        <font state="running">basefont_l_good</font>
+                        <font state="running">basefont_l_running</font>
                         <font state="warning">basefont_l_warn</font>
                         <font state="disabled">basefont_l_off</font>
-                        <alpha>100</alpha>
+                        <alpha>100</alpha>
                     </textarea>
                     <textarea name="title" from="baseitemtext">
                         <area>50,0,60%,100%</area>
+                        <template>%title%% (|progresspercent|%)%</template>
+                        <font state="running">basefont_l_running</font>
                         <font state="disabled">basefont_l_off</font>
-                        <alpha>100</alpha>
+                        <alpha>100</alpha>
+
                     </textarea>
                     <textarea name="timedate" from="basetext_m">
                         <area>60%,0,-5,100%</area>
                         <align>right,vcenter</align>
-                        <alpha>100</alpha>
+                        <alpha>100</alpha>
                         <font state="disabled">basefont_m_off</font>
                     </textarea>
-                    <progressbar name="buttonprogress2" from="baseprogress_xs">
-                        <shape name="progressimage">
-                            <fill color="#004b00" alpha="255"/>
-                        </shape>
-                    </progressbar>
                 </state>
                 <state name="selectedactive" from="active">
-                    <area>0,0,100%,58</area>
-                    <textarea name="card">
-                        <alpha>255</alpha>
-                    </textarea>
-                    <textarea name="title">
-                        <alpha>255</alpha>
-                    </textarea>
-                    <textarea name="timedate">
-                        <alpha>255</alpha>
-                    </textarea>
+                    <alpha>255</alpha>
                 </state>
             </statetype>
         </buttonlist>
@@ -527,7 +526,7 @@
                 <area>0,100%-120,100%,120</area>
                 <!-- left side -->
                 <group name="iconfacts" from="basescheduleiconfacts">
-                    <area>0,0,50%,170</area>
+                    <area>0,0,50%,120</area>
                 </group>
                 <!-- right side -->
                 <imagetype name="group" from="baseicon" depends="recgroup">
